cql3/statements/create_keyspace_statement.cc

```diff
@@ -139,6 +143,24 @@ future<> cql3::statements::create_keyspace_statement::grant_permissions_to_creat
     });
 }
 
+future<::shared_ptr<messages::result_message>>
+create_keyspace_statement::execute(service::storage_proxy& proxy, service::query_state& state, const query_options& options) {
+    return schema_altering_statement::execute(proxy, state, options).then([this] (::shared_ptr<messages::result_message> msg) {
+        bool multidc = service::get_storage_service().local().get_token_metadata().
```

Comment:
Nit: There's service::get_local_storage_service() that can be used instead of service::get_storage_service().local()

Reply:
fixd